src/scripts/infuse/infuse.html

```diff
@@ -45,7 +45,7 @@ <h1 translate="Infusion"></h1>
             <span> + {{ vm.wildcardMaterialCost | number }} <span class="currency"><img alt="{{ ::vm.items[vm.wildcardMaterialHash].itemName }}" title="{{ ::vm.items[vm.wildcardMaterialHash].itemName }}" ng-src="{{ vm.items[vm.wildcardMaterialHash].icon | bungieIcon }}"></span></span>
           </div>
         </div>
-        <button ng-if="vm.getAllItems" ng-disabled="vm.transferInProgress" ng-click="vm.transferItems()" translate="Infusion.TransferItems"></button>
+        <button ng-if="vm.getAllItems" ng-disabled="vm.transferInProgress" ng-click="vm.transferItems()" translate="Infusion.TransferItems" style="margin-top: 10px"></button>
```

Comment:
Also you appear to be using an older version as translate= should be ng-i18next=